Dockerfile

```diff
@@ -1,6 +1,8 @@
 ARG  DOCKER_REGISTRY
 ARG  BASE_IMAGE_TAG_DATE
 FROM $DOCKER_REGISTRY/kuali/tomcat7:java8tomcat7-ua-release-$BASE_IMAGE_TAG_DATE
+ARG  KUALICO_TAG
```

Comment:
Can this be moved up to be with the other ARG definitions?

Reply:
Heather, initially placed the ARG instruction at that location up with the others.  However the ARG instruction goes out of scope at the end of the build stage where it was defined.   The correct update to this was to add the pertinent ARG instruction after the FROM instruction.   The Jira ticket had some comments to also explain this behavior.  This was the reason this ticket took longer than it should have.

More info from vendor below:
https://docs.docker.com/engine/reference/builder/#arg
https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact

Reply:
Ah, fascinating! Will merge after all the commits are squashed.

Reply:
Ok Heather I think these are squashed correctly now.